metadata_backend/database/db_services.py

```diff
@@ -1,37 +1,35 @@
-"""Services that handle database connections. Implemented with MongoDB."""
+"""Services that handle database connections. Implemented with MongoDB.
+
+MongoDB client should be shared across the whole application, so it's created
+here as module level variable.
+
+Admin access is needed in order to create new databases during runtime.
+Default values are the same that are used in docker-compose file
+found from deploy/mongodb.
+"""
 
 import os
 from typing import Dict
 
 from pymongo import MongoClient, errors
 from pymongo.cursor import Cursor
 
+# Set up database client
```

Comment:
how about we have a config module, to properly manage config ?

Reply:
yeah, sounds reasonable. I'll add it